docs/role.docs.yaml

```
@@ -0,0 +1,99 @@
+paths:
+  /api/v1/role:
+    post:
+      security: [{ jwt: [] }]
+      summary: Admin set roles
+      tags: [Role and Permissions]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Role_Permissions"
+      responses:
+        "200":
+          description: ok.
+        "401":
+          description: not authorized
+        "404":
+          description: Not Found
+
+    get:
+      security: [{ jwt: [] }]
+      summary: Admin get all roles
+      tags: [Role and Permissions]
+      responses:
+        "200":
+          description: ok.
+        "401":
+          description: not authorized
+        "404":
+          description: Not Found
+
+    delete:
+      security: [{ jwt: [] }]
+      summary: Admin a role
+      tags: [Role and Permissions]
+      parameters:
+        - name: role_id
+          in: path
+          required: true
+          description: Provide role_id you want delete
+          schema:
+            type: string
+            format: int64
+            minimum: 1
+      responses:
+        "200":
+          description: ok.
+        "401":
+          description: not authorized
+        "404":
+          description: Not Found
+
+  /api/v1/role/assign/{user_id}:
```

based on route you have 
```suggestion
  /api/v1/role/update/{user_id}:
```